src/Corrade/Utility/Arguments.cpp

```diff
@@ -141,7 +141,7 @@ std::vector<std::string> Arguments::environment() {
     return list;
 }
 
-Arguments::Arguments(std::string prefix): _prefix{prefix + '-'} {
+Arguments::Arguments(const std::string& prefix): _prefix{prefix + '-'} {
```

Comment:
I did this because I thought the `operator+(string&&, char)` would avoid the copy, maybe, if there would be enough capacity in the original string (which it probably won't be, so this is a pessimization). And then I forgot the `std::move()` anyway. Twice. Argh.